.github/workflows/labeler.yml

```diff
@@ -0,0 +1,31 @@
+name: labeler
+
+on:
+  push:
+    branches:
+      - 'main'
+    paths:
+      - '.github/labels.yml'
+      - '.github/workflows/labeler.yml'
+  pull_request:
+    paths:
+      - '.github/labels.yml'
+      - '.github/workflows/labeler.yml'
+
+jobs:
+  labeler:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v4
+      -
+        name: Run Labeler
+        uses: crazy-max/ghaction-github-labeler@v5
```

Comment:
Since this is a third-party action, can we pin the version to the full length commit sha? 
https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions